src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -462,6 +462,24 @@
         az acr run -r MyRegistry https://github.com/Azure-Samples/acr-tasks.git -f build-hello-world.yaml --platform linux
 """
 
+<<<<<<< HEAD
+=======
+helps['acr pack'] = """
+type: group
+short-summary: Manage Azure Container Registry Tasks that use Cloud Native Buildpacks.
+"""
+
+helps['acr pack build'] = """
+type: command
+short-summary: Queues a quick build task that builds an app and pushes it into an Azure Container Registry.
+examples:
+  - name: Queue a build for the current directory with the default Oryx-based buildpack.
+    text: az acr pack build -r MyRegistry -t {{.Run.Registry}}/node-app:{{.Run.ID}} .
+  - name: Queue a build for the given GitHub repository with Heroku's buildpacks.
+    text: az acr pack build -r MyRegistry -t {{.Run.Registry}}/node-app:{{.Run.ID}} --pull --builder heroku/buildpacks:18 https://github.com/Azure-Samples/nodejs-docs-hello-world.git
+"""
+
+>>>>>>> 9feda7cfd4ddfe7a27a459eb1ef747fa979807d1
```

Comment:
There are git diff artifacts in your code. These need to be removed.

Reply:
Yes, not sure how they got there. I resolved the conflicts with vscode and for some reason it didn't remove them. It's fixed now.